Reflection_Engine/Modify/MakeStatic.cs

```diff
@@ -0,0 +1,70 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2019, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Reflection.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+
+namespace BH.Engine.Reflection
+{
+    public static partial class Modify
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Returns the specific type from a type that contains generic parameters")]
+        [Input("method", "The generic type")]
+        [Output("type", "The specific type constructed from the generic one")]
+        public static MethodInfo MakeStatic(this MethodInfo method)
```

Comment:
This method is not correct, because I was not really able to convert an instance method into a static method.. Thus, it is not really used. We may consider removing it altogether.

Reply:
Yes, I suppose you now have figured out why it cannot work after your work on the UI side.
I would suggest to remove this then. No point on leaving public methods that don't work in the code.